addons/website/static/src/components/dialog/page_properties.js

```diff
@@ -155,108 +154,122 @@ DuplicatePageDialog.template = xml`
 </WebsiteDialog>
 `;
 DuplicatePageDialog.props = {
-    onClose: Function,
-    close: Function,
+    onDuplicate: {type: Function, optional: true},
     pageId: Number,
+    close: Function,
```

Comment:
I don't understand why this line is moved by your commit? If it was to order by alphabetical order maybe but if it is for no reason at all, avoid creating useless diff :+1:

Reply:
I checked the DOC on Owl repo & /web code and there is nothing that confirmed the order of props... so I just restored the diff :+1: